dev/TabView/TabViewItem.cpp

```diff
@@ -176,6 +176,20 @@ void TabViewItem::OnHeaderPropertyChanged(const winrt::DependencyPropertyChanged
 
 void TabViewItem::OnPointerPressed(winrt::PointerRoutedEventArgs const& args)
 {
+    if (args.Pointer().PointerDeviceType() == winrt::PointerDeviceType::Mouse)
+    {
+        auto pointerPoint = args.GetCurrentPoint(*this);
+        if (pointerPoint.Properties().IsLeftButtonPressed())
+        {
+            auto isCtrlDown = (winrt::Window::Current().CoreWindow().GetKeyState(winrt::VirtualKey::Control) & winrt::CoreVirtualKeyStates::Down) == winrt::CoreVirtualKeyStates::Down;
+            if (isCtrlDown)
+            {
+                // Return here so the base class will not pick it up, but let it remain unhandled so someone else could handle it.
+                return;
```

Comment:
This will make it so ctrl+click just does nothing ever, even if we're not clicking on a selected item.  Is that intended?  It seems if we're just not wanting to honor the pattern of ctrl+click for selection modification that we're inheriting from ListView, but shouldn't we just act like a ctrl+click is the same as a regular click in that case?

Reply:
Good point. I added a check to only ignore it if the item is already selected.